### ISSUE TITLE
Add fallback for envrionment variables

### ISSUE THREAD

**Issue by jjcollinge** [lawrencegripper/traefik-appinsights-watchdog#9]
If the parameters are not found via flaeg, fallback and check environment variables - this will allow users to override environment variables in isolated environment specific parameters files without having to embed them in the ServiceManifest.xml. This will help avoid having the appinsights key checked into source control and support multiple environments.

**Comment by lawrencegripper** [comment]
Would this be something we could PR into flaeg rather than implement here?

**Comment by serbrech** [comment]
I don't think that fits flaeg. Flaeg does 1 thing : parse command line arguments. Handling multiple configuration sources sounds like another layer that could _use_ flaeg.
In fact I think Containous built exactly that : https://github.com/containous/staert 
except it doesnt support environment variable... yet :)

> Three native sources :
>     Command line arguments using flæg package
>     TOML config file using toml package
>     Key-Value Store using libkv and mapstructure packages
> **An Interface to add your own sources**
> 

I can spike that if you guys agree.

**Comment by serbrech** [comment]
linking to the request : https://github.com/containous/staert/issues/35

**Comment by jjcollinge** [comment]
Nice find @serbrech - would be great if we picked up that PR and finished of the work - assuming we're most of the way there.
